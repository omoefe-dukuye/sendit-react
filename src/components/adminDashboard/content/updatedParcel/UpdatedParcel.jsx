import React from 'react';
import PropTypes from 'prop-types';

const NewParcel = ({ parcel: {
  newDestination,
  newDistance,
  additionalPrice
} }) => (
  <div className='create-order__new-order'>
    <h3>Parcel Updated</h3>
    <ul>
      <li>New Destination: {newDestination}</li>
      <li>New Distance: {newDistance}</li>
      <li>Addtional Price: {additionalPrice}</li>
    </ul>
  </div>
);

NewParcel.propTypes = {
  parcel: PropTypes.object
};

export default NewParcel;
