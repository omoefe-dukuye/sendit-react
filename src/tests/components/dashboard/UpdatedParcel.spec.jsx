import React from 'react';
import { shallow } from 'enzyme';
import '@babel/polyfill';
import UpdatedParcel from '../../../components/adminDashboard/content/updatedParcel';

test('Should render static component', () => {
  const wrapper = shallow(<UpdatedParcel parcel={{
    "status": 200,
    "id": "35",
    "message": "parcel destination updated",
    "newDestination": "Ikeja, Nigeria",
    "newDistance": "4 km",
    "additionalPrice": "N 600"
}} />);
  expect(wrapper).toMatchSnapshot();
});
