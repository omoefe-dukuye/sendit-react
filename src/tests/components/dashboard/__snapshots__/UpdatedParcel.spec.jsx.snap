// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render static component 1`] = `
<div
  className="create-order__new-order"
>
  <h3>
    Parcel Updated
  </h3>
  <ul>
    <li>
      New Destination: 
      Ikeja, Nigeria
    </li>
    <li>
      New Distance: 
      4 km
    </li>
    <li>
      Addtional Price: 
      N 600
    </li>
  </ul>
</div>
`;
